docs/import/running-the-import.md

```diff
@@ -49,16 +49,16 @@ Clone the repository and build the package:
 git clone $REPOSITORY
 cd metadata-ws
 git checkout $BRANCH
-sed -ie "s|<username></username>|<username>${ORACLE_USERNAME}</username>|" settings.xml
-sed -ie "s|<password></password>|<password>${ORACLE_PASSWORD}</password>|" settings.xml
+sed -ie "s|<username></username>|<username>${ORACLE_SSO_USERNAME}</username>|" settings.xml
+sed -ie "s|<password></password>|<password>${ORACLE_SSO_PASSWORD}</password>|" settings.xml
 mvn -s settings.xml clean install -DskipTests=true
 ```
 
 Write the list of studies to import to `studies-to-import.txt`.
 
 Run the import:
 ```bash
-time bsub -M 20G -I java -Xmx18g \
+time bsub -M 20G -I java -Xmx18g -Rusage \
```

Comment:
Sorry I wasn't clear earlier... Rusage argument is for bsub as in -R "rusage[mem=20480]"

Reply:
Oops, sorry about that! `-Rusage` totally seemed like a flag which Java might have, so I didn't even check it :) Fixed in https://github.com/EBIvariation/metadata-ws/pull/86/commits/3f3755f4b2643a1f0190e53179842da217cbf607.